pkg/test/framework/components/echo/common/deployment/namespace.go

```diff
@@ -54,9 +54,9 @@ type EchoNamespace struct {
 	B echo.Instances
 	// Standard echo app to be used by tests
 	C echo.Instances
-	// Dual-stack echo app to be used by tests if running in dual-stack mode
+	// v4-only echo app to be used by tests if running in dual-stack mode
```

Comment:
The `echo` app named `D` is configured as a DualStack Service [here](https://github.com/istio/istio/blob/23fc23c669d0567a029e77fa0fa9ee5dcee094ac/pkg/test/framework/components/echo/common/deployment/echos.go#L287).

Reply:
That is what is being changed in this PR?

Reply:
Ack. Please update - https://github.com/istio/istio/blob/66ed818498c912146727127d590319cb6a77275d/tests/integration/pilot/gateway_test.go#L207